Samples/src/IntroToIQSharp/Notebook.ipynb

```diff
@@ -0,0 +1,778 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "PrepareRandomMessage?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Other commands ##"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `%who`\n",
+    "\n",
+    "`%who` returns the list of all local and workspace operations available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/json": "[\"CheckPlus\",\"HelloAndres\",\"HelloPi\",\"HelloQ\",\"Microsoft.Quantum.Samples.IsMinus\",\"Microsoft.Quantum.Samples.IsPlus\",\"PrepareRandomMessage\",\"SetMinus\",\"SetPlus\",\"TestPrepareQubits\"]",
+      "text/html": [
+       "<ul><li>CheckPlus</li><li>HelloAndres</li><li>HelloPi</li><li>HelloQ</li><li>Microsoft.Quantum.Samples.IsMinus</li><li>Microsoft.Quantum.Samples.IsPlus</li><li>PrepareRandomMessage</li><li>SetMinus</li><li>SetPlus</li><li>TestPrepareQubits</li></ul>"
+      ],
+      "text/plain": [
+       "CheckPlus, HelloAndres, HelloPi, HelloQ, Microsoft.Quantum.Samples.IsMinus, Microsoft.Quantum.Samples.IsPlus, PrepareRandomMessage, SetMinus, SetPlus, TestPrepareQubits"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%who"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `%package`\n",
+    "\n",
+    "`%package` allows you to load nuget packages and makes available any Q# operations defined on them. For example, to use the operations from [Q#'s Quantum Chemistry Library](https://docs.microsoft.com/en-us/quantum/libraries/chemistry/?view=qsharp-preview), you must load the [Microsoft.Quantum.Chemistry](https://www.nuget.org/packages/Microsoft.Quantum.Chemistry/) nuget package:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/json": "[\"Microsoft.Quantum.Canon::0.5.1903.2210\",\"Microsoft.Quantum.Chemistry::0.5.1903.2210\"]",
+      "text/html": [
+       "<ul><li>Microsoft.Quantum.Canon::0.5.1903.2210</li><li>Microsoft.Quantum.Chemistry::0.5.1903.2210</li></ul>"
+      ],
+      "text/plain": [
+       "Microsoft.Quantum.Canon::0.5.1903.2210, Microsoft.Quantum.Chemistry::0.5.1903.2210"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%package Microsoft.Quantum.Chemistry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`%package` returns the list of nuget packages currently loaded and their version."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `%version`\n",
+    "\n",
+    "`%version` simply returns the current IQ# and Jupyter Core versions:"
```

Comment:
It might be good to mention what Jupyter Core is here.
```suggestion
    "`%version` simply returns the current versions of IQ# and of Jupyter Core (a library used by IQ#):"
```